### PR TITLE
Replaces Station Administrator with Colony Director, NASA Approved

### PR DESCRIPTION
**Pull request by SpadesNeil**
Resolves https://github.com/PolarisSS13/Polaris/issues/2601

The discussion is over. NASA has input their opinion on the matter. Captain, Station Admin, Site Manager, Overseer, it's _all_ wrong!

Colony Director!

... Yes, I know Research Director is a thing, and I don't care. People will still call him Research Director. No one is getting Head of Personnel and Head of Security confused (even though they act like it sometimes). No one is going to confuse RD with CD. He'll be known simply as, "the Director." Does anyone call the RD "the Director"? Not as far as I know.

This sounds infinitely less awful than Station Administrator, which should be a CentCom role, and **not** a station role.

My Email:
> I'm trying to resolve a dispute among an online community who plays a sci-fi video game called Space Station 13, and I wanted to know what the title of an individual in charge of such a facility might be.
> 
> For context, the "station" we play on is based inside of an asteroid. It's technically an asteroid colony. Not a very big asteroid, but an asteroid all the same! People live and work on this asteroid colony.
> 
> Now previously, the title of the leader was simply "Captain," as it had been in previous versions of the game when the station was simply a space station like the ISS, but bigger. However, people have argued that name is incorrect because it's a term used in the military, and that NASA wouldn't use it in this context. I've argued this to be false, but I want to resolve once and for all what the correct term would be.
> 
> So, say NASA managed to build a long term or even permanent colony on an asteroid orbiting another world, be it Jupiter or otherwise. What would NASA call the man/woman in charge? The Captain, the Overseer, the Station/Colony Administrator, the Site Manager, or something else? Would the name change if the asteroid colony were no longer a satellite of a planet, and in the asteroid belt?

----------------------------------
NASA's reply:

> Greetings,
> 
> Thank you for your email and interest in NASA.
> 
> This is an intriguing question and I’ve sent some queries out to other offices. I have so far come up blank on formal agency policies ( http://nodis3.gsfc.nasa.gov/ ) dealing with this, although I do not have access to the Center policies of Johnson Space Center which trains and manages the astronaut programs.
> 
> Reviewing the crew descriptions from shuttle and ISS missions (and finding a confirming section [ http://history.nasa.gov/SP-4012v7ch3a.pdf ] within NASA Historical Data Book, Vol. VII: NASA Launch Systems, Space Transportation, Human Spaceflight, and Space Science, 1989-1998. NASA SP-4012, Washington, DC: NASA History Program Office, 2009), the ranks are as follows:
> ·         Commander
> ·         Pilot
> ·         Mission Commander
> ·         Mission Specialist
> ·         Payload Commander
> ·         Payload Specialist
> 
> In discussing your query with my team, we realized there could be a different analogue within the agency since your station would presumably have more than a half-dozen members – Center Management (for example, this is the Org Chart ( https://www.nasa.gov/sites/default/files/files/Center_Org_Chart_Visio_September2014.pdf ) for the Goddard Space Flight Center in greenbelt, MD). So, it this case, the title would be “Director”. At the agency level ( http://history.nasa.gov/orgcharts/orgchartNovember2015.pdf ), the top person would be the Administrator, followed by Deputy Administrator, then Associate Administrators operating the different offices. My suspicion on a large research facility such as you describe would be that the Center Management organization would be the more likely arrangement.
> 
> If I find formal policies, I will be sure to pass them along to you.
> 
> Thank you for an interesting query to end the day.
> 
> Yours,
> Rick Spencer
> Richard Spencer, Manager
> NASA Onliine Directive Information System(NODIS)- Library
> 
> Information Services Group, BFJV, Inc.
> Library, Archives, & Information Center
> NASA Headquarters
> 300 E St. SW 1W53
> Washington, DC 20546
> http://www.hq.nasa.gov/office/hqlibrary/

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Doop, forgot the changelog. One sec.

**Comment by SpadesNeil** [comment]
Fixed. Changelog added.

File: html/changelogs/SpadesNeil-ColonyDirector.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Anewbe) [inline html/changelogs/SpadesNeil-ColonyDirector.yml]: N3X15

Comment (SpadesNeil) [inline html/changelogs/SpadesNeil-ColonyDirector.yml]: jhasjklahgjazklghfjka

**Comment by SpadesNeil** [comment]
Fixed incorrect crediting.

File: html/changelogs/SpadesNeil-ColonyDirector.yml (hunk 1 above)
Comment (SpadesNeil) [inline html/changelogs/SpadesNeil-ColonyDirector.yml]: cant argue

**Comment by Anewbe** [comment]
Cool, this will be merged soonish.

**Comment by Signmark** [comment]
NASA-approved wizard removal when

**Comment by zaredman** [comment]
s8wnrxc

**Pull request merged**